src/test/resources/regressions/features/globals/globals-simple01.gobra

```diff
@@ -1,9 +1,11 @@
 // Any copyright is dedicated to the Public Domain.
 // http://creativecommons.org/publicdomain/zero/1.0/
 
-initEnsures acc(&A) && acc(&B) && acc(&C, 1/2)
+pkgInvariant acc(&A) && acc(&B) && acc(&C, 1/2)
 package pkg
 
+// ##(--experimentalFriendClauses)
```

Comment:
Not sure what's the problem here but the unit tests fail saying that `--experimentalFriendClauses` should be used